src/output.rs

```diff
@@ -62,10 +62,32 @@ mod tests {
 
 	#[test]
 	fn print_prints() {
-		// TODO!
 		//  add item to map
```

Comment:
We can get rid of all these comments in the future, too. I don't think they serve any value now that the code is here.